### ISSUE TITLE
Anchor Size Setting

### ISSUE THREAD

**Issue by bmankirLinker** [CVMI-Lab/ST3D#13]
For the `WAYMO->KITTI` adaptation, the size normalized model uses the custom object size for anchors (somewhere in between `WAYMO-KITTI`) even though the training data object size is reduced to `KITTI` and the test is the `KITTI` itself. But from a theoretical point of view, it should be used as `KITTI` mean object size in `secondiou/secondiou_old_anchor_sn.yaml`.
Would like to ask how does the author decides on those sizes or not to use the `KITTI` default?

**Comment by jihanyang** [comment]
We just use an old version of waymo anchor size in openpcdet. KITTI anchor size has also been attempted but no gains are found.

**Comment by bmankirLinker** [comment]
@jihanyang thanks for the answer. Moreover, according to my experiments, using no SN/ROS WAYMO trained model updated with KITTI anchor size (during eval) achieves similar results to SN-WAYMO pre-trained model's performance on KITTI test set. However, keeping the WAYMO anchor size reduces performance a lot.

**Comment by jihanyang** [comment]
We only try your second attempts "For the SN case changing the anchor size to KITTI mean didn't achieve anything for me as well".  I think your result is reasonable. Since the model just learn to regress the offset of prediction according to the anchor size,  changing the anchor size during eavluation is kind of similar to change the mean size of object.
